frontend/app/package.json

```diff
@@ -82,11 +82,11 @@
     "c8": "7.13.0",
     "cross-env": "7.0.3",
     "csv": "6.2.7",
-    "cypress": "12.6.0",
+    "cypress": "12.7.0",
     "dmg-license": "1.0.11",
     "dotenv": "14.3.2",
     "dotenv-cli": "5.1.0",
-    "electron": "22.3.0",
```

Comment:
can you rollback this to 22.x ? 

https://support.google.com/chrome/thread/185534985/sunsetting-support-for-windows-7-8-8-1-in-early-2023?hl=en